### ISSUE TITLE
Several TS stream video are shown in 4:3 format

### ISSUE THREAD

**Issue by matteogobbi** [imoreapps/ffmpeg-avplayer-for-ios-tvos#22]
Even if these streams are correctly visible with 16:9 format in other IPTV clients (i.e. VLC, m3u player, etc) here are shown in 4:3. How can I solve this problem?

**Comment by imoreapps** [comment]
Did you try the videoAspectRatio property of FFAVPlayerController component?

**Comment by matteogobbi** [comment]
Yes of course but it doesn't change. Channels that in other players are shown in 16/9, here are shown in 4/3 getting squized.
